### PR TITLE
Log config file validations so that they are picked up by monitoring

### PR DESCRIPTION
**Pull request by alistair-singh**

### CODE REVIEW

File: relayer/cmd/run/beacon/command.go [hunk 1]
```diff
@@ -62,7 +60,8 @@ func run(_ *cobra.Command, _ []string) error {
 
 	err = config.Validate()
 	if err != nil {
-		return fmt.Errorf("config file validation failed: %w", err)
```
Comment (alistair-singh) [inline relayer/cmd/run/beacon/command.go:65]: We only return errors from inner functions. This is the top level most function so we can safely log fatal here.

**Review by yrong — APPROVED** [review]
+1

**Review by claravanstaden — APPROVED** [review]
Nice catch!

**Comment by claravanstaden** [comment]
@alistair-singh what config file item was missing?

**Pull request merged**